python/src/nanoarrow/visitor.py

```diff
@@ -74,9 +75,60 @@ def to_columns(obj, schema=None) -> Tuple[List[str], List[Sequence]]:
     >>> names
     ['col1']
     >>> columns
-    [[1, 2, 3]]
+    [nanoarrow.c_lib.CBuffer(int64[24 b] 1 2 3)]
     """
-    return ColumnsBuilder.visit(obj, schema)
+    return ColumnsBuilder.visit(obj, schema, handle_nulls=handle_nulls)
+
+
+def nulls_forbid() -> Callable[[CBuffer, Sequence], Sequence]:
+    def handle(is_valid, data):
+        if len(is_valid) > 0:
+            raise ValueError("Null present with null_handler=nulls_forbid()")
+
+        return data
+
+    return handle
+
+
+def nulls_debug() -> Callable[[CBuffer, Sequence], Tuple[CBuffer, Sequence]]:
+    def handle(is_valid, data):
+        return is_valid, data
+
+    return handle
+
+
+def nulls_as_sentinel(sentinel=None):
+    from numpy import array, result_type
+
+    def handle(is_valid, data):
+        is_valid = array(is_valid, copy=False)
+        data = array(data, copy=False)
```

Comment:
```suggestion
        is_valid = np.array(is_valid, copy=False)
        data = np.array(data, copy=False)
```

Even though the import is just above, I was already confused about to what kind of array we were converting this, and looking for a `from nanoarrow import array` import at the top of the file ... So I think better to stick with explicit module (alias), which is also the common way numpy is used in Python

Reply:
Also, i am not entirely sure the `copy=false` is correct here for `is_valid`, or is that already unpacked at this point?